src/app/sign-up/sign-up.component.scss

```diff
@@ -0,0 +1,58 @@
+.sign-up-form-page-background {
+  height: 100%;
+  background: url("../../assets/auth-background.jpg") no-repeat;
+  background-size: cover;
+}
+
+.container {
+  max-width: 100rem;
+  padding: 0 1rem;
+  height: 100%;
+  margin-inline: auto;
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+.sign-up-form {
+  max-width: 32rem;
+  width: 100%;
+
+  &__title {
+    font-family: 'impact', sans-serif;
+    font-size: 2.625rem;
+    color: #F8BB15;
+    margin-bottom: 2.5rem;
+  }
+
+  @media (max-width: 25rem) {
```

Comment:
not sure it is a best approach to have rem in media query

Reply:
updated